### ISSUE TITLE
react-native link broken for iOS: links libBackgroundGeolocation.a instead of libRCTBackgroundGeolocation.a

### ISSUE THREAD

**Issue by GeovationMax** [mauron85/react-native-background-geolocation#231]
## Your Environment

* Plugin version: 0.5.0-alpha.35
* Platform: iOS
* OS version: 11.0
* Device manufacturer and model: Apple Simulator
* React Native version: 0.50.4
* Plugin configuration options: {debug: true}
* Link to your project: not public

## Context
After linking the native dependencies through react-native link the native dependencies are not available in iOS. When I call BackgroundGeolocation.configure I get a message that RNBackgroundGeolocation is undefined.

## Expected Behavior
`react-native link`  should create the links to native dependencies correctly so that the plugin works out of the box without further manual intervention

## Actual Behavior
`react-native link` links libBackgroundGeolocation.a instead of libRCTBackgroundGeolocation.a. This results in the native dependencies not being available

## Steps to Reproduce

1. create a react-native app
2. install react-native-background-geolocation
3. run `react-native link`
4. try to use the module in the javascript code, for example
```
import BackgroundGeolocation from 'react-native-mauron85-background-geolocation';

....

BackgroundGeolocation.configure({debug: true})
```

## Debug logs

```
TypeError: Cannot read property 'configure' of undefined

```

**Comment by mauron85** [comment]
Couldn't reproduce your issue. I've just tried:

1. react-native init myapp && cd myapp
2. yarn add react-native-mauron85-background-geolocation
3. react-native link react-native-mauron85-background-geolocation
4. import module and call configure and start in App.js
5. react-native run-ios

Env:
Plugin version: 0.5.0-alpha.35
React Native: 0.56
XCode: 9.3

I had to run `react-native run-ios` twice, because on first attempt it didn't build app correctly, but it was react-native or xcode issue.
